.github/CONTRIBUTING.md

```diff
@@ -31,70 +31,93 @@ If you at any point have any questions please read: [I have a question related t
+
+**Publish a WIP draft**: We encourage contributors to publish a draft pull-request with a WIP implementation of their issue to get feedback. This can advantageously be done before tests are written. By doing this misunderstandings and issues can be tackled early on; here they are often easier to correct. If this is not done, they might not be discovered until the code-review. Tests might have to be changed. So therefore we encourage contributors to use the opportunity to get some feedback. 
+
+<!-- Why do a self-review? -->
+**Self-review**: This consists of code-reviewing your own pull-request, following guidelines found in: [the good self review](#self-review). This is your chance to catch any low-hanging fruits and common issues before the reviewer does. If this is not done, an unneccessar amount of minor changes might be required during code review. This wastes both yours and the reviewers time. 
+
+<!-- Why we do UX reviews? -->
+**UX review**: If the implementation of the issue resulted in visual changes - then it has to be UX reviewed. This is to make sure that the introduced changes align with the design system. We strive to have every issue that introduces visual changes UX reviewed. If this is not done and the changes are merged to master, we might recieve issues about components not being properly aligned with the design system anymore from the users of Kirby. 
+
+Our experience is that pull-requests with a troublesome and drawn out review process has not followed one or more of these steps. 
+
+<!-- - Issues for trackability and history -->
 
 ### Before you get coding
-You should as a minimum skim through the "[the good](#the-good)"-section. This is because we are a lot of people working together on this project, all with varying prefrences both in terms of how we work and code. At the same time we strive for consistency in the Kirby codebase which might lead to some frustrations during code-reviews. In an effort to avoid this; we have attempted to specify our preferences on various subjects in that section. 
+Skim through the "[the good](#the-good)"-section. We are a lot of people working together on this project, all with varying prefrences both in terms of how we work and code. This might lead to frustrations during code-reviews. In an effort to avoid this and be transparent; we have specified our preferences on various subjects in that section. 
```

Comment:
Minor suggestion: 

Use capitalization for the link text. Should be done everywhere we use the heading or title of the section (or external page) the link refers to.

Examples:

* `Skim through the "[The good](#the-good)"-section` instead of `Skim through the "[the good](#the-good)"-section`
* `Make sure you have read: [Before you get coding]` instead of `Make sure you have read: [before you get coding]`

Reply:
Should be solved by new commits ☀️